### DISCUSSION TITLE
OR query when Filtering in Projects doesn't actually work

### DISCUSSION THREAD

**Discussion by fabis94** [community/community#33929] [Projects and Issues] [answered]
Here are the docs for how to use filters: https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/filtering-projects#filtering-items

There's this part that mentions how to use an OR query:
> To filter for any match of multiple values (an OR query), separate the values with a comma. For example label:"good first issue",bug will list all issues labelled good first issue or bug.

Well, the thing is, the OR query doesn't work! Maybe it works when only using `label`, but when I try to do something like `assignee: @me, reviewers: @me` it doesn't work!

Either the documentation should be updated to clarify in which scenarios does an OR query actually work, or better yet - the OR query should be fixed to support all possible filters

**Comment by alexanderniebuhr** [comment, marked as answer]
As far as I understood correctly, the OR only works for the same field item, while in all other filters, multiple fields are chained by an AND. Having the same issue :(

**Reply by 2percentsilk** [reply]
This is correct- OR only works within the field, not across fields. we'll work to update our [docs](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/filtering-projects#filtering-items) to make that more clear and hope to bring more advanced filtering to you.
